ci/build_python.sh

```diff
@@ -19,27 +17,52 @@ rapids-logger "Begin py build"
 
 CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
 
+RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION)
+export RAPIDS_PACKAGE_VERSION
+
+# populates `RATTLER_CHANNELS` array
+source rapids-rattler-channel-string
+
+rapids-logger "Prepending channel ${CPP_CHANNEL} to RATTLER_CHANNELS"
+
+RATTLER_CHANNELS=("--channel" "${CPP_CHANNEL}" "${RATTLER_CHANNELS[@]}")
+
 sccache --zero-stats
 
-# TODO: Remove `--no-test` flag once importing on a CPU
-# node works correctly
-RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION) rapids-conda-retry build \
-  --no-test \
-  --channel "${CPP_CHANNEL}" \
-  conda/recipes/cuml
+rapids-logger "Building cuml"
+
+# --no-build-id allows for caching with `sccache`
+# more info is available at
+# https://rattler.build/latest/tips_and_tricks/#using-sccache-or-ccache-with-rattler-build
+rattler-build build --recipe conda/recipes/cuml \
+                    --experimental \
+                    --no-build-id \
+                    --channel-priority disabled \
+                    --output-dir "$RAPIDS_CONDA_BLD_OUTPUT_DIR" \
+                    "${RATTLER_CHANNELS[@]}"
 
 sccache --show-adv-stats
+sccache --zero-stats
```

Comment:
```suggestion
```

This looks like it's unnecessary. It's wasted time on CUDA 11 builds (where no other compilation will happen after this line), and redundant with this existing call on CUDA 12 builds (where we compile `cuml-cpu`):

https://github.com/rapidsai/cuml/blob/84946a9d49fdcbaedbcbe4c70c5bec4de51ea48c/ci/build_python.sh#L37